### PR TITLE
Qt hmi

### PR DESCRIPTION
**Pull request by zenghuan1**
This PR is to fix some existing bugs.

**Risk**

This PR makes no RPC changes.

**Summary**

Fixed some bugs in Setting and Phone testing.
Adjustments made to open function modules Nav and Meida, prior to the initial words of SDLApps.

### CODE REVIEW

**Comment by tmacdwh1986** [comment]
could you list what bug fix is contained inside this change? It is hard to track if we don't have any records.Thanks.

**Comment by zenghuan1** [comment]
The bug function of this modification has nothing to do with SDL, it is not a major modification, and the bug is not mentioned on github.
I will add some comments to the details of github.

File: SDL_Qt_HMI/Phone/UI/CallingView.cpp [hunk 1]
```diff
@@ -35,13 +35,16 @@ void CallingView::viewAction(int state)
...
+            PhoneData::Inst()->SetAddNewCall(false);
+            UpdateData();
+            AddCall();
+            PhoneData::Inst()->SetCallingStatus(true);
+        }
         this->show();
         break;
     case eViewStatus_Inactive:
-        m_callTimer1.stop();
         break;
```
Comment (zenghuan1) [inline SDL_Qt_HMI/Phone/UI/CallingView.cpp]: Add Phone background voice call feature,remove the call stop time.

File: SDL_Qt_HMI/HMIWidgets/CComboBox.cpp [hunk 2]
```diff
@@ -43,11 +43,12 @@ void CComboBox::setItemDelegate(CComboBoxDelegate *delegate)
     delegate->SetItemSize(this->size());
 }
 
-void CComboBox::setBtnStyle(const QString &style,const QSize& size)
+void CComboBox::setBtnStyle(const QString &style,const QSize& size,int fontsize)
 {
     QString iwidth= QString::number(size.width());
```
Comment (zenghuan1) [inline SDL_Qt_HMI/HMIWidgets/CComboBox.cpp]: Expand AddComboBox interface font color setting function.

**Pull request merged**